### DISCUSSION TITLE
Dashboards displaying bar vs psi or km/h vs m/h

### DISCUSSION THREAD

**Discussion by 3rdStng** [CarlosCuezva/dashboards-Grafana-Teslamate#16] [Q&A] [answered]
First off, love the dashboards. They are a great enhancement to Teslamate's original ones. The "Top Drives" dashboard is displaying the average speed as km/h instead of mph. I don't know if this was a result of the latest update or not as I just loaded these dashboards a few days ago. Is there a way to change this or have it pull from Teslamate what my settings are?

Secondly, on the "Tire pressure" dashboard. I figured out how to change the left side from bar to PSI. (This was a new setting in Teslamate I glossed over) But the right side panel is still showing the values in bar. Is this also easy to change?

Thanks for the great dashboards!

**Comment by CarlosCuezva** [comment, marked as answer]
Hi 3rdStng.

Thanks to you and I hope you enjoy the dashboards.

**Tire Pressure**

There was a bug in the "Tire pressure in selected period" panel, I have uploaded a new version to fix it. Now displays data correctly in PSI or bar.

![image](https://user-images.githubusercontent.com/5815421/229197568-a06c6bc7-8f48-4f18-b51b-13dd3751eca1.png)

You would have to install this dashboard again to get the changes.

**Drive Tops**

Indeed, there was an error with the conversion between km and miles. It is already solved, you will have to re-install this dashboard again.

![image](https://user-images.githubusercontent.com/5815421/229204488-7af173e4-7eb9-4b66-a3e4-0e69e9b9d860.png)

Thank you very much for reporting these problems.

Regards

**Reply by 3rdStng** [reply]
Thank you! What a fast turn around too. Already loaded the new dashboards. What are your thoughts on a MPGe column somewhere? Or a Drive v2 dashboard? I know this is really an American thing, but I like to quickly say I got 160 MPGe on that trip. I posted about it over on Adrian's github, but I don't think it is going to go anywhere. https://github.com/adriankumpf/teslamate/discussions/2645

**Comment by CarlosCuezva** [comment]
Thanks to you.

I saw the conversation you mention, I had no intention of doing anything about it because it is very specific.

Now I'm going on vacation for a few days but when I come back I can make an adaptation.

**Reply by 3rdStng** [reply]
Awesome. It is pretty specific. I tried to do it myself, but I'm not a Grafana, or SQL/Query, expert and didn't know where to begin. I appreciate everything, even if the ultimate answer is no. 

Enjoy your vacation!

**Comment by CarlosCuezva** [comment]
I just updated the dashboards and included support for MPGe, I hope you like it

![image](https://user-images.githubusercontent.com/5815421/230038901-39945a93-cb5a-44be-82d8-7499ba4f6948.png)

**Reply by 3rdStng** [reply]
Yes. Thank you very much. No more math on my part!

**Comment by 3rdStng** [comment]
With the recent Tire Pressure dashboard update, the Current TP is displaying bars instead of PSI

![image](https://user-images.githubusercontent.com/24888063/232098956-1ae8ccb0-f050-4c61-b231-e4b4617f9f62.png)

**Reply by CarlosCuezva** [reply]
Arggggggggg sorry, I just corrected it.

Thanks for the notice
